<launch>
  <param name="output_frame_id" value = "map" />           
  <param name="output_pointcloud_topic" value = "/output_point" />
  <param name="input_pcd_path" value = "/home/eco/catkin_ws_pcd/pcd/test.pcd" />
  <param name="output_pcd_path" value = "/home/eco/catkin_ws_pcd/pcd/output.pcd" />

  <param name="crop_box_filter_enable" value = "true" />
  <param name="voxel_grid_filter_enable" value = "true" />
  <param name="remove_outliers_enable" value = "true" />
  <param name="rotate_translate_enable" value = "true" />


  <!-- crop_box_filter module -->
  <param name="crop_box_filter_min_x" value = "-100.0" />
  <param name="crop_box_filter_max_x" value = "100.0" />
  <param name="crop_box_filter_min_y" value = "-100.0" />
  <param name="crop_box_filter_max_y" value = "100.0" />
  <param name="crop_box_filter_min_z" value = "-1.0" />
  <param name="crop_box_filter_max_z" value = "30.0" />

  <!-- voxel_grid_filter module -->
  <param name="voxel_size_x" value = "0.1" />
  <param name="voxel_size_y" value = "0.1" />
  <param name="voxel_size_z" value = "0.1" />

  <!-- remove_outliers module -->
  <param name="sor_nearby_number" value = "10" />
  <param name="sor_thresh_value" value = "5.0" />

  <!-- rotate module -->
  <param name="rotate_value_x" value = "0.0" />
  <param name="rotate_value_y" value = "0.0" />
  <param name="rotate_value_z" value = "0.0" />

  <!-- translate module -->
  <param name="translate_value_x" value = "100.0" />
  <param name="translate_value_y" value = "200.0" />
  <param name="translate_value_z" value = "0.0" />

  <node pkg="pcd_tool" name="pcd_tool" type="pcd_tool" output="screen" />

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find pcd_tool)/rviz/pcd_tool.rviz" />
</launch>


